File: Python/Ex-2_Teams.py
# 
# Ex-2_Teams.py
# Created by Sebastian Jusca on 17/04/2023.
#

month = int(input('Enter the number of the month: '))
if month < 8:
	if month % 2 == 1: days = 31
	elif month == 2: days = 28
	else: days = 30
else:
	if month % 2 == 0: days = 31
	else: days = 30

print('Your month has {} days'.format(days))

File: C++/Ex-2_Teams.cpp
//
//  Ex-2_Teams.cpp
//  Created by Sebastian Jusca on 22/09/2023.
//

#include <iostream>
using namespace std;

int main(){
  int n, n_disp = 1, n_pari = 0;

  cout << "Enter value: " << endl;
  cin >> n;

  for(int i = 0; i<n; i++){
   cout << n_disp + n_pari << " ";
    n_pari+=2;
    n_disp+=2;
  }

  return 0;
}